proposals/0013-wave-size-range.md

```diff
@@ -83,11 +83,18 @@ Where `minWaveSize` is the minimum wave size supported by the shader
 
 ### DXIL Additions
 
-The existing `WaveSize` value is stored as a metadata 32-bit integer constant.
-To store the additional values, an additional metadata tag indicating a tuple of
- three 32-bit integers representing the given parameters to `WaveSize`.
-In the case where no preferred wave size is specified, the value zero will
- indicate that nothing was specified.
+The existing `WaveSize` value is stored as a single metadata 32-bit integer
+ constant.
```

Comment:
This is confusing.  It seems to imply that when only using the existing feature, there is a single metadata value.  Then, combined with wording below, you might be led to believe that we expand the same tag to 3 values.  I think it's a lot more clear if we can just explicitly say that we deprecate the `kDxilWaveSizeTag` (11) in SM 6.8 in favor of a new tag `kDxilRangedWaveSizeTag` (23), and here's the format of the value for that new tag...

Then, in the table below, if we could include both the old tag and the new tag, as well as a shader model column, it demonstrates that we still use the old tag for older shader models, and it has the same format there, but the new tag is always used for SM 6.8 and above.